src/screens/LoadWallet.js

```diff
@@ -95,6 +95,7 @@ class LoadWallet extends React.Component {
     // First we clean what can still be there of a last wallet
     wallet.generateWallet(this.state.words, '', pin, password, this.props.history);
     LOCAL_STORE.markBackupDone();
+    LOCAL_STORE.open(); // Mark this wallet as open, so that is does not appear locked after loading
```

Comment:
This was actually a bug discovered during the tests for this refactor: any imported wallet would be locked.

The steps to reproduce this are:
- Reset the wallet
- Close the app and open it again
- Import a wallet
- Bug: the wallet would be locked again immediately after successful history loading

Reply:
Do we find this behavior in other wallets, or is it specific for this wallet?

Reply:
It is specific to this wallet, because it has a special logic to display the "Welcome" screen.